### ISSUE TITLE
Dataset used for finetuning?

### ISSUE THREAD

**Issue by nawazia** [yuhoo0302/Segment-Anything-Model-for-Medical-Images#13]
Hi,

Just wondering how you finetuned SAM with COSMOS 1050K: "Thus, we finetuned SAM using part of the COSMOS 1050K to improve SAM’s perception of medical objects. Specifically, we considered 45 common and typical objects for finetuning the SAM."

I was wondering if you would be able to give me more detail on this subset; specifically, did you use any specific subsampling of COSMOS 1050K or just random selection?

**Comment by yuhoo0302** [comment]
Hi,

We used 45 common and typical objects (details about the objects refer to Fig. 18) to fine-tune the medical SAM.  The subset includes all image-label pairs in COSMOS 1050K that contain the specific objects (i.e., one of the selected 45 objects).  We have discussed with our clinical experts to choose them for fine-tuning evaluation (not random selection), and we hope the subset will be diverse and representative (e.g., different objects and modalities).

Thanks

**Comment by nawazia** [comment]
Great thanks a lot!
